Peoplesoft/main.tf

```diff
@@ -0,0 +1,538 @@
+  compute_hostname_prefix   = "${var.compute_hostname_prefix_es}"
+  compute_instance_count    = "${var.elastic_instance_count}"
+  vm_size                   = "${var.vm_size}"
+  os_publisher              = "${var.os_publisher}"   
+  os_offer                  = "${var.os_offer}"
+  os_sku                    = "${var.os_sku}"
+  os_version                = "${var.os_version}"
+  storage_account_type      = "${var.storage_account_type}"
+  compute_boot_volume_size_in_gb    = "${var.elastic_boot_volume_size_in_gb}"
+  data_disk_size_gb         = "${var.data_disk_size_gb}"
+  data_sa_type              = "${var.data_sa_type}"
+  admin_username            = "${var.admin_username}"
+  admin_password            = "${var.admin_password}"
+  custom_data               = "${var.custom_data}"
+  compute_ssh_public_key    = "${var.elastic_ssh_public_key}"
+  enable_accelerated_networking     = "${var.enable_accelerated_networking}"
+  vnet_subnet_id            = "${module.create_subnets.subnet_ids["elasticsearch"]}"
+  boot_diag_SA_endpoint     = "${module.create_boot_sa.boot_diagnostics_account_endpoint}"
+  create_av_set             = true 
+  create_public_ip          = false
+  create_data_disk          = false
+  assign_bepool             = false
+}
+
+###################################################
+# Create Process Scheduler server
+module "create_ps" {
+  source  = "./modules/compute"
+
+  resource_group_name       = "${azurerm_resource_group.rg.name}"
+  location                  = "${var.location}"
+  tags                      = "${var.tags}"
+  compute_hostname_prefix   = "${var.compute_hostname_prefix_ps}"
+  compute_instance_count   = "${var.prosched_instance_count}"
+  vm_size                   = "${var.vm_size}"
+  os_publisher              = "${var.os_publisher}"   
+  os_offer                  = "${var.os_offer}"
+  os_sku                    = "${var.os_sku}"
+  os_version                = "${var.os_version}"
+  storage_account_type      = "${var.storage_account_type}"
+  compute_boot_volume_size_in_gb    = "${var.prosched_boot_volume_size_in_gb}"
+  data_disk_size_gb         = "${var.data_disk_size_gb}"
+  data_sa_type              = "${var.data_sa_type}"
+  admin_username            = "${var.admin_username}"
+  admin_password            = "${var.admin_password}"
+  custom_data               = "${var.custom_data}"
+  compute_ssh_public_key    = "${var.prosched_ssh_public_key}"
+  enable_accelerated_networking     = "${var.enable_accelerated_networking}"
+  vnet_subnet_id            = "${module.create_subnets.subnet_ids["application"]}"
+  boot_diag_SA_endpoint     = "${module.create_boot_sa.boot_diagnostics_account_endpoint}"
+  create_av_set             = true 
+  create_public_ip          = false
+  create_data_disk          = false
+  assign_bepool             = false
+
+ 
+}
+###################################################
+# Create Tools Client machine
+
+module "create_toolsclient" {
```

Comment:
Need one of these for Identity VMs too. More info in this Whitepaper: https://cloud.oracle.com/iaas/whitepapers/deploy_peoplesoft_jdedwards_across_oci_azure.pdf

Reply:
Done.